Add direct link to auto discovery docs in mysql/postgres aurora setup (#23110)
DataDog/documentation: content/en/database_monitoring/setup_mysql/aurora.md

[Restart the Agent][3] to start sending MySQL metrics to Datadog.

> **The Datadog Agent supports auto-discovering all aurora endpoints in a cluster.** In order to do this, please [follow this guide.][10]

Reviewer: ```suggestion
**The Datadog Agent supports [Autodiscovery][10] of all Aurora endpoints in a cluster.** 
```

Simplifying this sentence so it flows better to just link directly within to the autodiscovery config page (also the link wasn't working as is).
